docs/docs/dev/policies/timeout.md

```diff
@@ -8,7 +8,52 @@ Specify the proxy to configure with the `sources` selector, and the outbound con
 
 The policy lets you configure timeouts for `HTTP`, `GRPC`, and `TCP` protocols.
```

Comment:
Worth mentioning protocol conf needs to be set in the config and link to the appropriate page.